apps/andi/lib/andi_web/input_schemas/data_dictionary_form_schema.ex

```diff
@@ -19,13 +19,12 @@ defmodule AndiWeb.InputSchemas.DataDictionaryFormSchema do
   def changeset(changes), do: changeset(%__MODULE__{}, changes)
 
   def changeset(dictionary, changes) do
-    source_format = Map.get(changes, :sourceFormat, nil)
     changes_with_id = StructTools.ensure_id(dictionary, changes)
     source_type = Map.get(changes, :sourceType, "ingest")
 
     dictionary
     |> cast(changes_with_id, [], empty_values: [])
-    |> cast_assoc(:schema, with: &DataDictionary.changeset(&1, &2, source_format), invalid_message: "is required")
```

Comment:
The dropdown for ingestions still allows for `csv`, `json`, `xml`, `geojson`, `zip archive`, and `GTFS Protobuf`.

Reply:
Oh, I misunderstood what you were asking. It accepts json or csv yes, regardless of the sourceFormat (since sourceFormat is no longer there)

https://github.com/UrbanOS-Public/smartcitiesdata/blob/master/apps/andi/lib/andi_web/live/edit_live_view/data_dictionary_form.ex#L95